lib/amplitude.dart

```diff
@@ -12,20 +12,36 @@ import 'package:amplitude_flutter/events/group_identify_event.dart';
 class Amplitude {
   Configuration configuration;
   MethodChannel _channel = const MethodChannel("amplitude_flutter");
+  /// Whether the Amplitude instance has been successfully initialized
+  ///
+  /// ```
+  /// var amplitude = Amplitude(Configuration(apiKey: "apiKey"));
+  /// // If care about init complete
+  /// await amplitude.isBuilt;
+  /// ```
+  late Future<bool> isBuilt;
```

Comment:
wdyt about returning the Amplitude instance on success instead of true? IT might make for easier usage in the cause customers want to wait e.g.

```
analytics = await Amplitude(...).isBuilt
```

Reply:
I'd rather keep it as `Future<bool>` instead of `Future<Amplitude?>` to avoid `?`. Customers can always achieve the goal (check whether the instance is initialized or not) by separating the call.

Reply:
Good call on the `?`. Current implementation makes sense.